internal/report/report.go

```diff
@@ -755,6 +755,15 @@ func printTraces(w io.Writer, rpt *Report) error {
 		}
 		sort.Strings(labels)
 		fmt.Fprint(w, strings.Join(labels, ""))
+
+		// Print any numeric labels for the sample
+		var numLabels []string
+		for k, v := range sample.NumLabel {
+			numLabels = append(numLabels, fmt.Sprintf("%10s:  %s\n", k, strings.Trim(fmt.Sprintf("%v", v), "[]")))
+		}
+		sort.Strings(numLabels)
```

Comment:
I'm sorting the keys -- like "bytes" and "requests". This makes sure I'll get bytes before requests.

Is my naming unclear?